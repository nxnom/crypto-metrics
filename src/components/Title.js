import React from 'react';
import PropTypes from 'prop-types';

import styles from './Title.module.css';

function Title({ children }) {
  return <h3 className={styles.title}>{children}</h3>;
}

Title.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Title;
