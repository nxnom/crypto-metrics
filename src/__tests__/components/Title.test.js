import { render } from '@testing-library/react';

import Title from 'src/components/Title';

describe('Title', () => {
  it('renders correctly', () => {
    const { container } = render(<Title>Title</Title>);
    expect(container).toMatchSnapshot();
  });
});
