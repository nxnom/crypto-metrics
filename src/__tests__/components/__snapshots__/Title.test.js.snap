// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div>
  <h3
    class="title"
  >
    Title
  </h3>
</div>
`;
